internal/access/organization.go

```diff
@@ -44,6 +48,25 @@ func CreateOrganization(c *gin.Context, org *models.Organization) error {
 	return data.CreateOrganization(db, org)
 }
 
+func GenerateOrganizationDomain(orgName string, serverHostname string) string {
+	replaceSymbols := func(r rune) rune {
+		switch {
+		case unicode.IsLetter(r), unicode.IsNumber(r):
```

Comment:
This will return true for some characters that may not be URL safe in all scenarios. 

Ex:
```
unicode.IsLetter('ð') // true
```

This may work but also presents some possible complications. I think its safer to follow [RFC2396](https://www.ietf.org/rfc/rfc2396.html#section-2.3).
```
   alpha    = lowalpha | upalpha

   lowalpha = "a" | "b" | "c" | "d" | "e" | "f" | "g" | "h" | "i" |
              "j" | "k" | "l" | "m" | "n" | "o" | "p" | "q" | "r" |
              "s" | "t" | "u" | "v" | "w" | "x" | "y" | "z"

   upalpha  = "A" | "B" | "C" | "D" | "E" | "F" | "G" | "H" | "I" |
              "J" | "K" | "L" | "M" | "N" | "O" | "P" | "Q" | "R" |
              "S" | "T" | "U" | "V" | "W" | "X" | "Y" | "Z"

   digit    = "0" | "1" | "2" | "3" | "4" | "5" | "6" | "7" |
              "8" | "9"

   alphanum = alpha | digit
```

Reply:
True, I cut some corners here because our validation for this field should exclude all other characters. But that validation could change without someone realizing to update this function, so it would be good to be more strict.

I'll change this to `r >= 'a' && r <= 'z'`, etc

Reply:
I made those changes